### DISCUSSION TITLE
How can I buy the app if not using Play services?

### DISCUSSION THREAD

**Discussion by Diapolo** [d4rken-org/sdmaid-se#623] [Q&A] [answered]
I'm using CalyxOS, which comes with microG and Aurora Store, but I can't seem to buy a license for your new app. How can I get a licensed version? Thanks for your hard work.

**Comment by d4rken** [comment, marked as answer]
Use GitHub releases:
https://github.com/d4rken-org/sdmaid-se/releases

and sponsor development if possible:
https://github.com/sponsors/d4rken

**Reply by Diapolo** [reply]
Thanks for that, will this version give update information for new releases?
And what would be the current price for buying it on play store?

**Reply by d4rken** [reply]
> Thanks for that, will this version give update information for new releases?

Yes there is an update check, you will see a message when opening the app, if there is a newer version.

> And what would be the current price for buying it on play store?

That depends on the country, I don't know this by heart, sorry.

**Reply by Diapolo** [reply]
Is there any chance to send some money via IBAN and BIC? I don't want to hint Github my personal data.

**Reply by d4rken** [reply]
If neither Google Play nor Sponsoring is an option then just use GitHub releases without sponsoring. Maybe send some money to a charity :slightly_smiling_face:.
